This would crash on runtime as LTL doesn't support options with the name same,
even if they are identical.

--- fitstools/cosmicfits.cpp	2018-10-18 19:24:20.318275732 +0200
+++ fitstools/cosmicfits.cpp	2018-10-18 19:24:37.362337623 +0200
@@ -661,9 +661,6 @@
       flags.addOption(new FloatOption("sky_offset", "0.0",
                                       "already subtracted constant sky surface, needed to calculate correct noise estimation from surface fit and gain",
                                       0, &flagVal.sky_offset) );
-      flags.addOption(new RegionArrayOption("region", "0:0,0:0",
-                                            "x_start:x_end,y_start:y_end only filter region",
-                                            'r', 2, &flagVal.region) );
       flags.addOption(new StringOption("prefix", "c", "output file prefix",
                                        'p', &flagVal.prefix) );
       flags.addOption(new BoolOption("mask", "FALSE",
